src/app/DTT/time-zone/index.js

```diff
@@ -0,0 +1,21 @@
+import { Select } from '@chakra-ui/react'
+import { map } from 'ramda'
+import defaultTimeZones from 'utils/helpers/time-zone.json'
```

Comment:
why are we using a json here? just breaks the pattern of js objects that we normally use

Reply:
I saw the JSON, leave it as a JSON it is fine